spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/config/DataFlowClientProperties.java

```diff
@@ -37,7 +37,7 @@
 	/**
 	 * Skip Ssl validation.
 	 */
-	private boolean skipSslValidation = true;
+	private boolean skipSslValidation;
```

Comment:
I thought we wanted to enable SSL validation by default.

Reply:
We should never "skip ssl validation" by default. I am not sure how that got set to `true` in the first place.